openedx/features/job_board/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.conf.urls import url
 
-from .views import list_jobs, show_job_detail
+from .views import add_job, list_jobs, show_job_detail
 
 
 urlpatterns = [
-    url(r'^jobs/list/$', list_jobs, name='job_list'),
-    url(r'^jobs/list/[0-9]$', list_jobs, name='job_detail'),
```

Comment:
Please rebase with imran/covid19, As it will have merge conflicts with abdullah zafar's work.